### PR TITLE
feat: mdx code tabs

### PR DESCRIPTION
**Pull request by kellyjosephprice**
| [![PR App][icn]][demo] | Fix RM-9016 |
| :--------------------: | :---------: |

## 🧰 Changes

Adds support for code tabs in MDX!

Adds a custom transformer and compiler to support code tabs. Also refacors the processors to make adding plugins a little easier, maybe?

In reviewing tests, I noticed a bunch that don't apply anymore. So they've been deleted.

Also, to get the visual tests working, I added `remark-gfm` so we have basic support for tables now?

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

File: docs/code-blocks.md [hunk 1]
```diff
@@ -50,73 +45,63 @@ We offer both classic single code blocks, as well as a tabbed interface for disp
 }
 ```
 
-<div id="EXAMPLE-CUSTOM-CODE">
-
+<div style={{ '--md-code-background': '#e3dcef', '--md-code-text': '#4a2b7b', '--md-code-tabs': '#c6b8dd', '--md-code-radius': '4px' }}>
```
Comment (kellyjosephprice) [inline docs/code-blocks.md:48]: CSS doesn't work in JSX, and I'm not sure what method of styling we should promote?

**Comment by kellyjosephprice** [comment]
Sorry, this isn't ready or review yet.

File: components/Code/index.tsx [hunk 2]
```diff
@@ -27,74 +26,39 @@ function CopyCode({ codeRef, rootClass = 'rdmd-code-copy', className = '' }) {
...
-  lang: '',
-  meta: '',
-};
-
-CreateCode.sanitize = sanitizeSchema => {
-  // This is for code blocks class name
-  sanitizeSchema.attributes.code = ['className', 'lang', 'meta', 'value'];
-
-  return sanitizeSchema;
-};
```
Comment (jennspencer) [inline components/Code/index.tsx:98]: oohh do we not need this sanitize step anymore

Comment (kellyjosephprice) [reply components/Code/index.tsx:98]: Pretty sure. It was to prevent code execution, and we're kind of throwing that out the window.

**Comment by kellyjosephprice** [comment]
Sorry, forgot I didn't have code tabs working inside another block.

**Review by jennspencer — APPROVED** [review]
could probs nix the dollar signs in `Code/index.tsx` and fix up the TS errrs but otherwise looks & works A+

**Comment by kellyjosephprice** [comment]
> fix up the TS errrs

I was just hoping they'd go away on there own /s

File: docs/code-blocks.md (hunk 1 above)
Comment (jennspencer) [reply docs/code-blocks.md:48]: this would be a good Q for the sync tomorrow

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.75.0-beta.28</b>](/readmeio/markdown/releases/tag/v6.75.0-beta.28)
<br>

**Pull request merged**